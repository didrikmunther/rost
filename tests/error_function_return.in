fn b() -> int {
	return 1;
}

fn main() {
	let a: pointer<int> = b();
}