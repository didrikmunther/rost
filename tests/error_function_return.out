  --> ./tests/error_function_return.in:6:24 => CompilerError
5 | fn main() {
6 |     let a: pointer<int> = b();
  |                           ^^^
  |                           └─ Wrong type: int, expected: *int
7 | }